From b295e457130ffe07802ce79013d08b6551bd2416 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Wed, 19 Aug 2015 13:36:26 -0500
Subject: [PATCH 092/131] axxia: Temporary Work-Around for a Hardware Issue

Only boot 8 cores for now until hardware works.

Signed-off-by: John Jacques <john.jacques@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/intel/axc6712-cpus.dtsi | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/axc6712-cpus.dtsi b/arch/arm64/boot/dts/intel/axc6712-cpus.dtsi
index f351e2f..ed0c4f5 100644
--- a/arch/arm64/boot/dts/intel/axc6712-cpus.dtsi
+++ b/arch/arm64/boot/dts/intel/axc6712-cpus.dtsi
@@ -70,34 +70,5 @@
 			enable-method = "psci";
 
 		};
-
-		cpu@8 {
-			device_type = "cpu";
-			compatible = "arm,armv8";
-			reg = <0x400>;
-			enable-method = "psci";
-		};
-
-		cpu@9 {
-			device_type = "cpu";
-			compatible = "arm,armv8";
-			reg = <0x401>;
-			enable-method = "psci";
-		};
-
-		cpu@10 {
-			device_type = "cpu";
-			compatible = "arm,armv8";
-			reg = <0x402>;
-			enable-method = "psci";
-		};
-
-		cpu@11 {
-			device_type = "cpu";
-			compatible = "arm,armv8";
-			reg = <0x403>;
-			enable-method = "psci";
-
-		};
 	};
 };
-- 
2.8.1

